### ISSUE TITLE
Will you be presenting your work on certain platforms, and when?

### ISSUE THREAD

**Issue by LYMDLUT** [MasterBin-IIAU/UNINEXT#4]
Will you be presenting your work on certain platforms, and when?

**Comment by MasterBin-IIAU** [comment]
Yes, we have plans to introduce UNINEXT on certain platforms. When the detailed timeline is determined, we will let you know by replying to this issue.

**Comment by LYMDLUT** [comment]
Maybe openmmlab is a good choice
